geode-core/src/main/java/org/apache/geode/internal/tcp/Connection.java

```diff
@@ -513,7 +516,7 @@ protected Connection(ConnectionTable t, Socket socket)
     this.isReceiver = true;
     this.owner = t;
     this.socket = socket;
-    this.conduitIdStr = owner.getConduit().getSocketId().toString();
+    this.conduitIdStr = conduit.getSocketId().toString();
```

Comment:
where did `owner` go?

Reply:
we don't need owner here since we have the conduit as a parameter